static/i18n/en/app.yml

```diff
@@ -309,17 +311,17 @@ settings:
     passwordDesc: Enhance your privacy. Set a password to encrypt Ledger Live data stored on your computer, including account names, balances, transactions and public addresses.
     changePassword: Change password
     sync: Synchronize accounts
-    syncDesc: Resynchronize your accounts with the network.
+    syncDesc: Resynchronize your accounts with the network
     export: Export logs
-    exportDesc: Exporting Ledger Live logs may be necessary for troubleshooting purposes.
+    exportDesc: Exporting Ledger Live logs may be necessary for troubleshooting purposes
     softResetTitle: Clear cache
-    softResetDesc: Clear the Ledger Live cache to force resynchronization with the blockchain.
+    softResetDesc: Clear the Ledger Live cache to force resynchronization with the blockchain
     softReset: Clear
     hardResetTitle: Reset Ledger Live
-    hardResetDesc: Erase all Ledger Live data stored on your computer, including your profile, accounts, transaction history and settings. The private keys that manage your crypto assets remain secure on your Ledger device.
+    hardResetDesc: Erase all Ledger Live data stored on your computer, including your accounts, transaction history and preferences. The private keys that manage your crypto assets remain secure on your Ledger device.
```

Comment:
charles mentionned it should be at the end something like: `The private keys that manage your crypto assets remain secure on your Ledger device and on your Recovery sheet.`

Reply:
Good point, will add.